examples/hybrid-compute/batched_wasm_example.py

```diff
@@ -0,0 +1,303 @@
+# # Quantinuum Hybrid Compute via pytket
+# This notebook contains an example of how to create and run hybrid classical-quantum workflows within the Quantinuum stack via `pytket`.
+# **Note:** This notebook uses existing Wasm examples provided in this folder's subdirectories. To create and run your own Wasm functions, you'll need to set up an environment on your machine to support this workflow. Instructions for how to do this are given in this folder's README.
+# * [Repeat Until Success](#repeat-until-success)
+# This is a simple example made of how you can process measurements, using classical logic via Wasm, to implement a quantum loop, or a repeat-until-success (RUS) style circuit, also called an exit-on-failure circuit.
+# * [1 RUS Circuit](#1-rus-circuit)
+# * [RUS Experiment](#rus-experiment)
+# ## 1 RUS Circuit 
+# ### Import libraries
+# Note the `wasm` module imported from `pytket` as well as several several conditional operators that are options. More information on the conditional operations available can be found in the user manual at [Classical and Conditional Operations](https://tket.quantinuum.com/api-docs/classical.html).
+
+
+from pytket.extensions.quantinuum import QuantinuumBackend
+
+machine = "H1-1E"
+backend = QuantinuumBackend(device_name=machine)
+backend.login()
+
+# Set up the RUS circuit. This involves 2 steps:
+# 1. Set up Wasm File handler. This checks the Wasm file and can list the available functions within the file.
+# 2. Set up the pytket circuit, which calls the Wasm file handler.
+
+import pathlib
+from pytket import wasm
+
+rus_dir = pathlib.Path().cwd().joinpath("RUS_WASM")
+wasm_file = rus_dir.joinpath("rus_wasm.wasm")
+wfh = wasm.WasmFileHandler(wasm_file)
```

Comment:
```suggestion
wfh = wasm.WasmFileHandler("rus_wasm.wasm")

Reply:
You can also do this, if the files is next to the notebook, maybe that is easier if porting the notebook somewhere else?